src/Timer_Obscure_Behaviour.md

```diff
@@ -1,41 +1,67 @@
 # Timer obscure behaviour
 
+<!-- TODO: fix the wording even more -->
+
+:::warning Use of names referring to DIV
+
+When detailing relations in the paragraphs below, 
+ DIV is just the visible part of the system counter.
+
+The system counter is constantly incrementing every M-cycle
+ while the CPU is not in STOP mode.
```

Comment:
```suggestion
:::warning System counter

DIV is just the visible part of the **system counter**.

The **system counter** is constantly incrementing every M-cycle, unless the CPU is in [STOP mode](<#Using the STOP Instruction>).
```

Reply:
I don't know if it's easily understandable, but it seems reasonable to me. (At some point, we have to hazard guesses.) "System counter" isn't set in stone IMO, as I'd rather come up with new terms if they're clearer and feasible to change (unlike e.g. hardware reg names); but this name clears the clarity bar IMO.

And indeed, a notice/tip would be preferable to a warning; amended the suggestion accordingly.

Reply:
Seems reasonable to me as well. I also think a standard "tip" type fits better than the warning.